lib/ansible/plugins/action/net_config.py

```diff
@@ -72,10 +72,13 @@ def _get_working_path(self):
             cwd = self._task._role._role_path
         return cwd
 
-    def _write_backup(self, host, contents):
-        backup_path = self._get_working_path() + '/backup'
+    def _write_backup(self, host, contents, path=None):
```

Comment:
Assuming path comes from the module args, you can access the path information from the instance variable `_task_args` so no need to pass it here